ecosystem/sep-0040.md

```diff
@@ -0,0 +1,160 @@
+## Preamble
+
+```
+SEP: To Be Assigned
+Title: Non-Fungible Token Standard
+Author: George Kudrayvtsev <george@stellar.org>
+Track: Standard
+Status: Draft
+Created: 2021-01-31
+Updated: 2021-01-31
+Version: 0.1.0
+Discussion: TBD
+```
+
+## Simple Summary
+This SEP defines a standard interface to describe non-fungible tokens (NFTs) on the Stellar network, including a distribution mechanism and metadata fields. It draws inspiration from the [EIP-721 standard](https://eips.ethereum.org/EIPS/eip-721).
+
+
+## Motivation
+A standard interface for NFTs allows ecosystem applications (such as wallets or marketplaces) to create, view, or otherwise work with any NFT on Stellar.
```

Comment:
This section could be more explicit in that it is motivated by interoperability, for an NFT created in one application, to be viewable, tradable, useable, anywhere Stellar is used.
```suggestion
A standard interface for NFTs that makes NFTs issued by any ecosystem application (such as a wallet or marketplace) accessible and interoperable with any other ecosystem application supporting NFTs.
```